app/models/pet_product.rb

```diff
@@ -0,0 +1,6 @@
+class PetProduct < ApplicationRecord
+  belongs_to :pet
+  belongs_to :product
+
+  validates_presence_of :notes
```

Comment:
Same as the pet model, we don't want to require the user to post a note if they don't want to.

Reply:
Another good point. I think I left this in there because I at first forgot to add it to the migration and this was how I ended up catching it in the final checklist. Removed and should be good to go.